pokete_classes/achievements.py

```diff
@@ -2,14 +2,18 @@
 
 import datetime
 import logging
-import scrap_engine as se
 from pokete_general_use_fns import liner
 from .hotkeys import ACTION_DIRECTIONS, Action, get_action
 from .loops import std_loop, easy_exit_loop
 from .ui_elements import BetterChooseBox, LabelBox
 from .color import Color
 from .notify import notifier
 
+try:
+    import scrap_engine as se
+except ImportError:
+    print("You seem to not have the 'scrap-engine' package installed")
+
```

Comment:
Yeah this was kind of a test I forgot to remove. I thought it would be nice to fetch all the required packages on startup first and see if all are available, otherwise a message with all the missing packages will be outputted. But I wasn't able to implement this behavior yet :o